CHANGES.rst

```diff
@@ -3,12 +3,15 @@
 
 New Features
 ^^^^^^^^^^^^
++ Continued fleshing out of the documentation.
```

Comment:
Don't think we need this. The new stuff has documentation but that should be the norm going forward.

Reply:
Noted.